x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/description.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiDescribedFormGroup, EuiFormRow, EuiLink } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { useMlKibana } from '../../../../../contexts/kibana';
+
+export const Description: FC = memo(({ children }) => {
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+  const docsUrl = docLinks.links.ml.customUrls;
```

Comment:
@szabosteve is there a page in the data frame analytics docs we could link to on custom URLs related to DFA jobs? This page is specific to anomaly detection. The functionality is similar, but just wondering if we should enhance the docs here?

Reply:
@peteharverson There is no specific docs page in the DFA section about custom URLs. I'll open an issue to track creating the page and address it soon. (Will add the link to the issue later.)